csrc/pybind.cpp

```diff
@@ -81,4 +81,16 @@ PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
     "get_device_attribute",
     &get_device_attribute,
     "Gets the specified device attribute.");
+
+  pybind11::module custom_ar = m.def_submodule("custom_ar", "custom allreduce");
```

Comment:
one nit - I'd be descriptive here and call this `custom_allreduce` - it's not obvious what `ar` stands in the general context.

Reply:
The module has no use outside `custom_all_reduce.py`, so I'll skip this one